build.gradle

```diff
@@ -391,12 +397,15 @@ subprojects { subproj ->
 	}
 
 	spotless {
-		def headerFile = rootProject.file('src/spotless/' + licenseHeaderFileProvider.call(project))
+		def headerText = rootProject.file('src/spotless/' + licenseHeaderFileProvider.call(project))
+		def importOrderConfig = rootProject.file('src/eclipse/junit-eclipse.importorder')
+		def formatterJavaConfig= rootProject.file('src/eclipse/junit-eclipse-formatter-settings.xml')
```

Comment:
Hi @fvgh, I think there should be a space in between `formatterJavaConfig` and the `=` sign. :)